src/components/Authorization/Login.tsx

```diff
@@ -20,7 +20,7 @@ export const LoginBox: React.FC = () => {
         if(window && (token !== "" && token !== undefined)) { 
             window.localStorage.setItem("token", JSON.stringify(token));
```

Comment:
Token here has still default value "" because component hasn't rerendered yet. You have to remove if statement beacuse it won't work anyway. Instead save some other info in localstorage like: token = true so we can access it.